dspace-server-webapp/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -3025,12 +3025,12 @@ public void testEntityTypePublication() throws Exception {
         Community community = CommunityBuilder.createCommunity(context)
             .withName("Parent Community")
             .build();
-        Collection collection = CollectionBuilder.createCollection(context, community).withName("Collection").build();
+        Collection collection = CollectionBuilder.createCollection(context, community).withName("Collection")
```

Comment:
Although it's good to test the new scenario, it's not ok that the current setup is no longer tested

Can you restore the original tests, and add these as separate new tests instead?